routes/web.php

```diff
@@ -16,3 +16,9 @@
 Route::get('/', function () {
```

Comment:
Replace the closure with a proper PageController

Reply:
Ok, working on it!